src/porepy/numerics/ad/grid_operators.py

```diff
@@ -186,7 +187,7 @@ def _error_message(self):
         return "Argument should be a subdomain grid or a list of subdomain grids"
 
 
-class MortarProjections(Operator):
+class MortarProjections:
```

Comment:
Please motivate change.

Reply:
I also removed the same parent from `BoundaryProjection`, and did so for `MortarProjections` just for the unity. These projection classes are never (as for my knowledge) used as operators directly - they don't have the `parse` method, so it won't work anyway. Instead, they are used as a collection of several projection operators, each of which is a proper ad operator, so there is no reason to make it an operator itself. Please, let me know if I'm wrong.

Reply:
Your argument makes sense to me. Do you agree, @keileg?

Reply:
Agree, this is a change in the right direction.